src/engine-actions/cancel-swap.js

```diff
@@ -0,0 +1,13 @@
+const { cancelInvoice } = require('../lnd-actions')
+
+/**
+ * Cancels the invoice for a swap
+ *
+ * @param {string} swapHash - hash of the sawp
```

Comment:
misspelled swap. Also, is this supposed to be a Base64 string?

Reply:
I guess because of the implicit conversion you mentioned it could probably work as either Bytes/Buffer or base64 string but we should try to choose one convention, which I believe is the Bytes/Buffer one? There are other places in the codebase that use `string` so we could update those later.